docker/Dockerfile

```diff
@@ -20,8 +20,8 @@ ARG AMD_ARCH=AMD_GFX90A
 ARG CUDA_ARCH=AMPERE80
 ARG CUDA_INSTALLER=https://developer.download.nvidia.com/compute/cuda/12.2.2/local_installers/cuda_12.2.2_535.104.05_linux.run
 ARG GCC_VERSION=11
-ARG KOKKOS_VERSION=4.2.00
-ARG LIGHTNING_VERSION=v0.34.0
+ARG KOKKOS_VERSION=4.3.01
```

Comment:
Would you like to trigger the docker action to check the support?

Reply:
Good call.

Reply:
tried to build, but couldn't get it to pull the PR version. We can try in a follow up